core/schemata/json-schema/service/service.json

```diff
@@ -0,0 +1,46 @@
+{
+    "$schema": "https://json-schema.org/draft/2019-09/schema",
+    "$id": "service",
+    "type": "object",
+    "title": "General Service Data",
+    "required": [
+      "serviceType"
+    ],
+    "unevaluatedProperties": false,
+    "allOf": [
+      {
+        "properties": {
+          "serviceType": {
+            "$id": "serviceType",
+            "type": "string",
+            "title": "Service Type",
+            "description": "The general type of the service",
+            "enum": [
```

Comment:
@setaman let us just rethink what a Service type could be. We will be bound to this if we do not want to write migrations ;)